### PR TITLE
Handle validation message

### PR DESCRIPTION
**Pull request by slevertbiot**

### CODE REVIEW

File: udmif/event-handler/src/InvalidMessageError.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export class InvalidMessageError extends Error {
+    constructor(msg: string) {
+        super(msg);
+
+        // Set the prototype explicitly.
+        Object.setPrototypeOf(this, InvalidMessageError.prototype);
+    }
+}
```
Comment (mspratt-biot) [inline udmif/event-handler/src/InvalidMessageError.ts:8]: I'm surprised a lot of these files have no new lines...lends me to believe husky isn't running...can you confirm?

File: udmif/event-handler/src/UdmiMessageHandler.ts [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
-      throw new Error('An invalid device name or id was submitted');
+      throw new InvalidMessageError('An invalid device name or id was submitted');
     }
 
     return { name: message.attributes.deviceId, id: message.attributes.deviceNumId };
   }
 
   private messageCanBeHandled(message: UdmiMessage): boolean {
-    return isPointsetSubType(message) || isSystemSubType(message);
+    return isPointsetSubType(message) || isSystemSubType(message) || (isValidationSubType(message) && message.attributes.deviceId !== '_validator');
```
Comment (mspratt-biot) [inline udmif/event-handler/src/UdmiMessageHandler.ts:39]: Should '_validator' be moved out to a const?

File: udmif/event-handler/src/__tests__/model/Validation.spec.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        }).toThrow(MESSAGE_ERROR);
+    })
+
+    test.each([undefined, null])('throws exception when timestamp is not provided', (timestamp: string) => {
+        expect(() => {
+            builder.timestamp(timestamp);
+        }).toThrow(TIMESTAMP_ERROR);
+    })
+
+    test.each([undefined, null])('throws exception when %p errors are not provided', (error: Error[]) => {
```
Comment (mspratt-biot) [inline udmif/event-handler/src/__tests__/model/Validation.spec.ts:34]: Just confirming if we want %p in this sentence?

**Review by mspratt-biot — APPROVED** [review]

File: udmif/event-handler/src/InvalidMessageError.ts (hunk 1 above)
Comment (slevertbiot) [reply udmif/event-handler/src/InvalidMessageError.ts:8]: I did not see Husky run

**Review by mspratt-biot — APPROVED** [review]

File: udmif/event-handler/src/InvalidMessageError.ts (hunk 1 above)
Comment (mspratt-biot) [reply udmif/event-handler/src/InvalidMessageError.ts:8]: Not sure if its setup in there. If it's not and I find myself in there one day I'll set it up.

**Pull request merged**